sw/device/silicon_creator/lib/drivers/flash_ctrl.h

```diff
@@ -111,6 +111,23 @@ rom_error_t flash_ctrl_read_start(uint32_t addr, uint32_t word_count,
  */
 size_t flash_ctrl_fifo_read(size_t word_count, uint32_t *data_out);
 
+typedef enum flash_ctrl_exec {
+  kFlashCtrlExecDisable = 0x5,
+  kFlashCtrlExecEnable = 0xa,
+} flash_ctrl_exec_t;
```

Comment:
We should align with https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/base/multibits.h
 
And consider if it makes sense to add more types in addition to `multi_bit_bool_t`.